docs/area-owners.md

```diff
@@ -63,75 +63,75 @@ Note: Editing this file doesn't update the mapping used by the `@msftbot` issue
 | area-System.Net.Security                       | @karelz       | @dotnet/ncl                                         |                                                                                                                                                                                                                                                                                           |
 | area-System.Net.Sockets                        | @karelz       | @dotnet/ncl                                         |                                                                                                                                                                                                                                                                                           |
-| area-System.Numerics                           | @jeffhandley  | @tannergooding                                      |                                                                                                                                                                                                                                                                                           |
-| area-System.Numerics.Tensors                   | @jeffhandley  | @tannergooding                                      |                                                                                                                                                                                                                                                                                           |
+| area-System.Numerics                           | @jeffhandley  | @michaelgsharp @tannergooding                       |                                                                                                                                                                                                                                                                                           |
+| area-System.Numerics.Tensors                   | @jeffhandley  | @michaelgsharp @tannergooding                       |                                                                                                                                                                                                                                                                                           |
 | area-System.Reflection                         | @jeffhandley  | @buyaa-n @joperezr @steveharter                     |                                                                                                                                                                                                                                                                                           |
```

Comment:
Seems like you didn't update the LEAD on the Reflection areas.

Reply:
actually seems like most of our pod areas still show you as the lead as opposed to @ericstj. Let me know if you want me to push a change to fix it.

Reply:
```suggestion
| area-System.Reflection                         | @ericstj      | @buyaa-n @joperezr @steveharter                     |                                                                                                                                                                                                                                                                                           |
```

Reply:
Good catch; thanks!!